index.html

```diff
@@ -0,0 +1,263 @@
+                </ul>
+            </div>
+      </div>
+  </section>
+
+  <section class="section1">
+          <ul class="section-wrap">
+               <li class="work">            
+                  <div class="work-container">
+                    <ul>
+                      <li>
+                          <img src="image/portfolio1.png" alt="Several hands holding beer glasses">
+                      <li class="work-description">
+                        <h1><a href="">Tonic</a></h1>
+                        <div class="work-info">
+                                <ul>
+                                  <li class="intro">Canopy</li>
+                                  <li> <i class="fa fa-circle"></i> Back End Dev</li>
+                                  <li><i class="fa fa-circle"></i> 2015</li>
+                                </ul>
+                                </div>
+                        <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+
+                        <div class="tags">
+                          <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                          </ul>
+                        </div>
+
+                        <a href="#" class="btn-one">See project</a>
+                      </li>
+                    </ul>
+                  </div>
+               </li>
+
+                <li class="work">            
+                  <div class="work-container">
+                    <ul>
+                      <li>
+                          <img src="image/portfolio2.png" alt="Several hands holding beer glasses">
+                      <li class="work-description">
+                        <h1><a href="">Multi-Post Stories</a></h1>
+                        <div class="work-info">
+                            <ul>
+                              <li class="intro">FACEBOOK</li>
+                              <li> <i class="fa fa-circle"></i> Full Stack Dev</li>
+                              <li><i class="fa fa-circle"></i> 2015</li>
+                            </ul>
+                        </div>
+                        <p>
+                          Experimental content creation feature that allows users to add to an 
+                          existing story over the course of a day without spamming their
+                          friends.
+                        </p>
+
+                        <div class="btn-tags">
+                         <ul>
+                            <li>html</li>
+                            <li>Ruby on Rails</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                          </ul>
+                        </div>
+
+                        <a href="#" class="btn-one">See project</a>
+                      </li>
+                    </ul>
```

Comment:
- [ ] I recommend you to use ul and li only for project technologies, please modify your code to keep to semantic code like section and include your heading and paragraph

-------------------Original ------------------
![design1](https://user-images.githubusercontent.com/55635977/126050346-7dfc05a9-df30-4f45-8493-10a352e1b0ac.png)
-------------------Yours ------------------
  
![design2](https://user-images.githubusercontent.com/55635977/126050359-d0ebfe9b-d57a-4732-9f3b-7544a4c2e089.png)

Kindly, refer to the figma to fix that issue, arrange your project technologies as flex not block or listed.

- [ ] The button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design) Please refer to the image below to implement the changes

![buttons](https://user-images.githubusercontent.com/55635977/126050408-2cd19215-8a8c-47fd-927c-4e59c052ec9f.png)